lib/config/flat-config-schema.js

```diff
@@ -13,7 +13,7 @@
  * Note: This can be removed in ESLint v9 because structuredClone is available globally
  * starting in Node.js v17.
  */
-const structuredClone = require("@ungap/structured-clone").default;
+const structuredClone = require("@ungap/structured-clone").default || require("@ungap/structured-clone");
```

Comment:
Since the error only seems to occurs in the browser test bundle which is not published, we could also try to use the `structuredClone` global first if it's defined. All current browsers support `structuredClone` as a built-in.

```suggestion
const structuredClone = globalThis.structuredClone || require("@ungap/structured-clone").default;
```